src/cipher.jl

```diff
@@ -89,7 +89,11 @@ end
 
 function trim_padding_PKCS5(data::Vector{UInt8})
   padlen = data[sizeof(data)]
-  return data[1:sizeof(data)-padlen]
+  if all(data[end-padlen+1:end] .== data[end])
```

Comment:
I guess technically, we want to check `data[end-padlen+1:end-1]`, since we don't need to check that `data[end] == data[end]` :P

Reply:
Changed!